go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
I see, thanks for clarifying! So V1 proto is not as capable as V2 proto.
Sorry for insisting on this, but I think we should not have an graph configuration API that work only for cloud training but not for local training. I am open for discussion, I think the graph configuration API needs to be consistent: works both locally and on cloud.
Maybe when we need to capability of v2 proto (e.g., different optimizer on different parameters), we should make it work locally as well.